libnmstate/nm/nmclient.py

```diff
@@ -51,6 +51,17 @@ def client(refresh=False):
     if _nmclient is None or refresh:
         if NM:
             _nmclient = NM.Client.new(None)
+            # if get_state() returns 0 the NM state is unknown
+            if _nmclient.get_state() == 0:
+                logging.error(
+                    'NetworkManager is in unknown state, '
+                    'please make sure NetworkManager is installed and running.\n'
+                    'Check the documentation for more information.'
+                )
+                # TODO - adjust documentation accordingly if needed
```

Comment:
We usually add the needed changes as part of the same PR or open an issue to track this missing part.
`TODO` is reserved for things we need to do at this part of the code so others will notice it is not complete and help fix. I guess this needs to be added to the user manual, so I see no need for a TODO here.

Reply:
Thanks for the info. I was not sure about the workflow so I put it in order to avoid any oversight.
I removed that in the new [commit b04e4b5](https://github.com/nmstate/nmstate/commit/b04e4b5d0135520d5925683c7da46ba1415ebd71).